ml/Database.cc

```diff
@@ -66,13 +93,22 @@ bool Statement::bindValue(size_t Pos, const std::string &Value) {
     return false;
 }
 
-bool Statement::bindValue(size_t Pos, const int Value) {
+/*bool Statement::bindValue(size_t Pos, const int Value) {
```

Comment:
why did you comment this one out?

Reply:
As explained in Slack, there would be ambiguity on the overloaded function binValue(,). Also, the one with const int argument does not seem to be used. But in a more effective code, we should have both and catch this ambiguity another way, I agree. atm it works, and no function needs bindValue with const int.